coordinator/src/services/translation/request/practitioner.ts

```diff
@@ -157,7 +157,8 @@ export function getAgentPersonTelecom(
 function convertAgentPersonPerson(
   practitionerRole: fhir.PractitionerRole,
   practitioner: fhir.Practitioner,
-  getAgentPersonPersonIdFn = getAgentPersonPersonIdForAuthor) {
+  getAgentPersonPersonIdFn = getAgentPersonPersonIdForAuthor
+) {
```

Comment:
formatting here is a little weird maybe but I'd quite like this function to have an explicit return type

Reply:
done